apps/console/src/init/init.ts

```diff
@@ -20,6 +20,7 @@ import { UAParser } from "ua-parser-js";
 import { AppUtils } from "./app-utils";
 import "core-js/stable";
 import "regenerator-runtime/runtime";
+import TimerWorker from "worker-loader!./worker";
```

Comment:
Since we have the `worker-loader`, do we really need this import syntax.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1994/commits/999c09a98747235d1f180660e2092d0c55fa0e82